e2e/playwright-ci.config.js

```diff
@@ -28,7 +29,9 @@ const config = {
     ignoreHTTPSErrors: true,
     screenshot: 'only-on-failure',
     trace: 'on-first-retry',
-    video: 'off'
+    video: 'off',
+    // @ts-ignore - custom configuration option for nyc codecoverage output path
+    coveragePath: fileURLToPath(new URL('../.nyc_output', import.meta.url))
```

Comment:
intellisense complains without this since coveragePath is a custom option not a part of the PlaywrightConfig type